software/wmla120.py

```diff
@@ -1923,7 +1928,8 @@ def _update_software_vars(self):
                     print(msg)
                     ch, _dir = get_selection(dirs)
                 else:
-                    self.log.error(f'No {repo_id} repo found in software server.')
+                    if self.running != 'prep':
+                        self.log.error(f'No {repo_id} repo found in software server.')
```

Comment:
From a maintenance perspective I suggest you add this to debug and put it in logs. Also, while this software does not have much use from outside users it is always beneficial to keep useful information in logs(errors, cmds, warnings) in the event debug is needed.

Reply:
done